tests/core/test_dialect.py

```diff
@@ -571,7 +571,7 @@ def test_model_normalization_multiple_serde(
     expressions = parse(
         f"""
         MODEL (
-            name {table},
+            name {exp.maybe_parse(table, into=exp.Table).sql(dialect=normalization_dialect)},
```

Comment:
This needed to be changed because when the normalization dialect is `mysql`, we produce `db."TABLE"` and double quotes actually denote strings. So we first parse using the base dialect and generate the proper identifier quotes per dialect.